.github/workflows/auto-approve-dependabot.yml

```diff
@@ -0,0 +1,27 @@
+# Automatically approve PRs made by Dependabot
+#
+# Written to look at the original author of the PR (instead of the current
+# actor) in order to be able to backresolve existing PRs using this action (by
+# mass labeling them). Leads to slightly unnecessary spammage of aprovals in a
+# PR...
+#
+# Only does approvals! A different GitHub Action takes care of merging.
+name: Auto-approve Dependabot
+on:
+  pull_request:
+    types:
```

Comment:
Should we also filter to only run when the PR has a `dependencies` label?

Reply:
I suppose we could, but why? This is the only thing dependabot does.

Reply:
There is not even a filter supported here. So the solution would be to change the `if` expression to this:

```
if: (github.event.pull_request.user.login == 'dependabot[bot]' || github.event.pull_request.user.login == 'dependabot-preview[bot]') && contains(github.event.pull_request.labels.*.name, 'dependencies')
```

Would you prefer that?